Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'

Upgrading from 0.1.5 to 0.2.0 causes the above error to be generated when running in a server environment which does not use babel.

Error dump:
```
Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /project/node_modules/graphql-passport/lib/index.js
- /project/server/authentication.js
- /project/index.js
    at Object.<anonymous> (/project/node_modules/graphql-passport/lib/index.js:3:30)
    at Generator.next (<anonymous>)
    at Object.<anonymous> (/project/server/authentication.js:1)
    at Generator.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/project/node_modules/graphql-passport/lib/index.js',
    '/project/server/authentication.js',
    '/project/index.js'
  ]
}
```

That is, from where my server code is attempting to set up passport auth using a GraphQLLocalStrategy object, the node module lib file fails. Looking at the source for index.js:

```js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "buildContext", {
  enumerable: true,
  get: function get() {
    return _buildContext["default"];
  }
});
Object.defineProperty(exports, "GraphQLLocalStrategy", {
  enumerable: true,
  get: function get() {
    return _GraphQLLocalStrategy["default"];
  }
});

var _buildContext = _interopRequireDefault(require("./buildContext"));

var _GraphQLLocalStrategy = _interopRequireDefault(require("./GraphQLLocalStrategy"));
```

Clearly, this is depending on babel for doing things.

My server environment isn't currently using babel, and I'd rather not go to the trouble of setting up babel just for this strategy. Downgrading 0.1.5 seems to work as expected for now, but I'd like to ensure I have bug fixes and new features as they become available.

Is babel, going forward, a dependency? If so, the readme and package.json should probably be updated to reflect that.

(I cannot, of course, rule out that _I'm_ doing something weird on my end which caused this, but, again, no babel in the server environment. About the only thing I can think of which might be causing this would be esm.) 

Method to reproduce:

In a fresh project:

```
mkdir test && cd test
yarn init --yes
yarn add graphql-passport
```

No other dependencies, the generated `node_modules/graphql-passport/lib/index.js` file has the babel dependency.

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,5 @@
+__mocks__
+.vscode
+node_modules
+.git*
+README.md

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "graphql-passport",
-  "version": "0.1.5",
+  "version": "0.2.0",
   "description": "Strategy and utilities to use passport.js with GraphQL server",
   "main": "lib/index.js",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "Strategy and utilities to use passport.js with GraphQL server",
   "main": "lib/index.js",
   "scripts": {
-    "build": "npm run clean && babel src -d lib --ignore src/**/*.test.js",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/**/*.test.js --ignore __mocks__",
     "clean": "rimraf lib",
     "lint": "eslint src",
     "prepublish": "npm run lint && npm run test && npm run build",

